src/lib/Zikula/Bundle/CoreInstallerBundle/Util/ControllerUtil.php

```diff
@@ -79,25 +76,23 @@ public function initPhp()
 
     public function requirementsMet(ContainerInterface $container)
     {
+        // several other requirements are checked before Symfony is loaded.
+        // @see app/SymfonyRequirements.php
+        // @see \Zikula\Bundle\CoreInstallerBundle\Util\ZikulaRequirements::runSymfonyChecks
         $results = [];
 
         $x = explode('.', str_replace('-', '.', phpversion()));
         $phpVersion = "$x[0].$x[1].$x[2]";
         $results['phpsatisfied'] = version_compare($phpVersion, \Zikula_Core::PHP_MINIMUM_VERSION, ">=");
 
-        $results['datetimezone'] = ini_get('date.timezone');
         $results['pdo'] = extension_loaded('pdo');
-        $results['phptokens'] = function_exists('token_get_all');
         $results['mbstring'] = function_exists('mb_get_info');
```

Comment:
So `$results['mbstring'] = function_exists('mb_get_info');` could be removed also then.